### ISSUE TITLE
[BUG] when running zenml init after installing 

### ISSUE THREAD

**Issue by EwanValentine** [zenml-io/zenml#99]
**Describe the bug**

**To Reproduce**
1. pyenv global 3.8.2
2. python -m venv venv
3. source venv/bin/activate.fish 
4. pip install zenml
5. zenml init

**Expected behavior**

Project to be initiated. 

**Stack Trace**

```
Traceback (most recent call last):
  File "/Users/ewanvalentine/work/zenml/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 584, in _build_master
    ws.require(__requires__)
  File "/Users/ewanvalentine/work/zenml/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 901, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Users/ewanvalentine/work/zenml/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 792, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (numpy 1.21.2 (/Users/ewanvalentine/work/zenml/venv/lib/python3.8/site-packages), Requirement.parse('numpy~=1.19.2'), {'tensorflow'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ewanvalentine/work/zenml/venv/bin/zenml", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/Users/ewanvalentine/work/zenml/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3253, in <module>
    def _initialize_master_working_set():
  File "/Users/ewanvalentine/work/zenml/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3236, in _call_aside
    f(*args, **kwargs)
  File "/Users/ewanvalentine/work/zenml/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 3265, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/Users/ewanvalentine/work/zenml/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 586, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/Users/ewanvalentine/work/zenml/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 599, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/Users/ewanvalentine/work/zenml/venv/lib/python3.8/site-packages/pkg_resources/__init__.py", line 792, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (numpy 1.21.2 (/Users/ewanvalentine/work/zenml/venv/lib/python3.8/site-packages), Requirement.parse('numpy~=1.19.2'), {'tensorflow'})

```

** Context (please complete the following information):**
 - OS: [e.g. Ubuntu 18.04] OSX 11.5.2
 - Python Version: 3.8.2
 - ZenML Version: latest

**Additional information**

I've tried Python 3.7.3, 3.9.3 and currently 3.8.2. I've tried to manually resolve some of the version conflicts by installing specific versions. But there seemed to be several, some I couldn't work out how to resolve.

Advice is much appreciated!

**Comment by htahir1** [comment]
This is the second issue that popped up today. We need to deeper investigation on it. Will get back to this thread latest by tomorrow

**Comment by EwanValentine** [comment]
Awesome, much appreciated @htahir1 🙌🏻

**Comment by htahir1** [comment]
@EwanValentine we seem to have a critical release failure this time around (darn it). Still investigating but might be delays. Will update this tread as it develops.

**Comment by bcdurak** [comment]
Hey @EwanValentine, as @htahir1 mentioned, we are investigating the issue at the moment and we will keep this thread updated.

As a temporary solution, I have tested the installation with two clean versions of Python, 3.6.9 and 3.8.2 respectively, which seems to solve the version conflicts and I was able to initialize a ZenML repository. Let us know if it works for you as well or if there are any other issues.
